aten/src/ATen/cuda/detail/CUDAHooks.cpp

```diff
@@ -64,6 +66,49 @@ Device CUDAHooks::getDeviceFromPtr(void* data) const {
   return at::cuda::getDeviceFromPtr(data);
 }
 
+c10::Stream CUDAHooks::getCurrentCUDAStream() const {
+  return c10::cuda::getCurrentCUDAStream();
```

Comment:
FYI this one is in DeviceGuardImpl already

Reply:
Updated. See the change in input_metadata.h.